homeassistant/components/updater/binary_sensor.py

```diff
@@ -20,6 +20,7 @@ def __init__(self):
         self._update_available = None
         self._release_notes = None
         self._newest_version = None
+        self._current_version = None
```

Comment:
There is no need to make it an instance variable or check if it exists, it is always there.

Reply:
Done.